_posts/2022-11-07-relicensing.md

```diff
@@ -0,0 +1,200 @@
+as a whole (namely ``import MDAnalysis`` or subclassing). This includes
+closed / commercial licenses (although we encourage the use of open
+source licenses). However, one would not be able to just take parts of
+the MDAnalysis code and put it into another piece of code unless the
+other code is then *also* licensed under a compatible copyleft license
+(i.e. GPLv3+/LGPLv3+).
+
+We considered other popular licenses but none fulfilled our requirements.
+
+   
+## How will the relicensing process work?
+
+As of writing, MDAnalysis has [over 160 contributors][contributors].
+All of which have contributed code under the terms of the GPL v2+
+license. We also have a large user community which uses the library
+for many wonderful scientific applications, including several
+downstream libraries.
+
+Ultimately, the final decision on relicensing [rests with code
+authors](#Contacting-contributors).  However, we fully recognise that
+this is a big change for the MDAnalysis user base and the wider
+molecular sciences community. We are, as always, fully invested in
+ensuring that our actions reflect the needs of our community. We
+therefore want to give everyone an opportunity to [ask questions about
+or comment on the relicensing effort](#Consultation-period) as part of
+this process.
+
+
+### Consultation period (7th November until 5th December 2022)
+
+We will start the process with a 28 days open consultation period.
+
+During this period we would like to encourage members of the community,
+both developers and users, to comment on and ask questions about the
+proposed relicensing efforts. The aim is to ensure that relicensing is
+indeed in the interest of the community. We will do our best to account
+for any concerns raised before attempting to continue with the long and
+time-consuming process of relicensing.
+
+We wish to open this converstation on our [public forums]({% link
+index.md %}#participating) (mailing lists, discord, twitter). As legal
+matters, such as licensing, can sometimes be sensitive in nature we have
+also set up an email address (*licensing@mdanalysis.org*) monitored
+solely by the MDAnalysis Core Developers for any private queries which
+you may have.
+
+_Note: Whilst the consultation will only last 28 days, we will continue
+to engage with conversations on this topic for the entire length of the
+relicensing process._
+
+
+### Contacting contributors (6th December onwards)
+
+After the consultation period, we will contact every code contributor to
+the core MDAnalysis library with a request to agree to changing their
+contributitions' license from the current "GPL v2 or any later version"
+to "LGPL v3 or any later version".
+
+It is important that we hear back from as many contributors as possible.
+If you have contributed to MDAnalysis in the past but have since changed
+your git-linked contact details, we would kindly ask if you could email
+*licensing@mdanalysis.org* to let us know how best to contact you.
+
```

Comment:
Are we setting expectations about what may constitute a successful majority?

Reply:
We very very much need not to, do you interpret this as setting an expectation @hmacdope ?

Reply:
No not at all, just querying because there were no expectations  stated. So if no expectations required goal achieved.

Reply:
Ok yeah, this is indeed intentional. I very much tried to ensure we don't trap ourselves in some promise we can't fulfill. The conditions for completion of the relicense are nuanced and cannot be laid out until after we've completed things (it's complicated and I won't get into more details here).

Reply:
:+1: :100: